web/src/layout/Header/navbar/Menu/Settings/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Dispatch, SetStateAction, useRef, useState } from "react";
+import styled from "styled-components";
+import { Tabs } from "@kleros/ui-components-library";
+import General from "./General";
+import SendMeNotifications from "./SendMeNotifications";
+import { useFocusOutside } from "hooks/useFocusOutside";
+
+const tabsItems = [
+  {
+    text: "General",
+    value: 0,
+  },
+  {
+    text: "Notifications",
+    value: 1,
+  },
+];
+
+interface ISettings {
+  setIsSettingsOpen: Dispatch<SetStateAction<boolean>>;
+}
+
+const Settings: React.FC<ISettings> = ({ setIsSettingsOpen }) => {
+  const [currentTab, setCurrentTab] = useState<number>(0);
+  const containerRef = useRef(null);
+  useFocusOutside(containerRef, () => setIsSettingsOpen(false));
+
+  return (
+    <Container ref={containerRef}>
+      <StyledSettingsText>Settings</StyledSettingsText>
+      <StyledTabs
+        currentValue={currentTab}
+        items={tabsItems}
+        callback={(n: number) => {
+          setCurrentTab(n);
+        }}
+      />
+      {currentTab === 0 ? <General /> : <SendMeNotifications />}
+    </Container>
+  );
+};
+
+const Container = styled.div`
+  display: flex;
+  position: absolute;
+  z-index: 1;
+  background-color: ${({ theme }) => theme.whiteBackground};
+  flex-direction: column;
+  border: 1px solid ${({ theme }) => theme.stroke};
+  border-radius: 3px;
+`;
+
+const StyledSettingsText = styled.div`
+  display: flex;
+  justify-content: center;
+  font-size: 24px;
+  color: ${({ theme }) => theme.primaryText};
+  margin-top: 24px;
+`;
+
+const StyledTabs = styled(Tabs)`
+  padding-left: 32px;
+  padding-right: 32px;
```

Comment:
let's make the padding and the width dynamic i.e. `padding: 0 calc(64px + (98 - 64) * (100vw - 300px) / (1250 - 300));` but with correct values